chains/substrate/config.go

```diff
@@ -1,39 +1,54 @@
 package substrate
 
 import (
-	"strconv"
+	"fmt"
+	"math/big"
 
 	"github.com/ChainSafe/chainbridge-core/chains"
+	"github.com/spf13/viper"
 )
 
 type SubstrateConfig struct {
 	GeneralChainConfig chains.GeneralChainConfig
-	StartBlock         uint64
+	StartBlock         *big.Int
 	UseExtendedCall    bool
 }
 
-func ParseConfig(rawConfig *chains.RawChainConfig) *SubstrateConfig {
-	config := &SubstrateConfig{
-		GeneralChainConfig: rawConfig.GeneralChainConfig,
-		StartBlock:         0,
-		UseExtendedCall:    false,
+type RawSubstrateConfig struct {
+	GeneralChainConfig chains.GeneralChainConfig
+	StartBlock         int64 `mapstructure:"startBlock"`
+	UseExtendedCall    bool  `mapstructure:"useExtendedCall"`
+}
+
+func GetConfig(path string, name string) (*SubstrateConfig, error) {
+	var config RawSubstrateConfig
```

Comment:
Consider using the convention we talked about to create a pointer to a newly initialized struct, especially if you are going to return this same data type so that the pointer is not forgotten later (+ see below comment)

```go
config := &RawSubstrateConfig{}
```